src/services/two-factor-registration-confirmer.js

```diff
@@ -1,7 +1,7 @@
 const P = require('bluebird');
 const request = require('superagent');
-const ServiceUrlGetter = require('./service-url-getter');
 const logger = require('./logger');
+const context = require('../context');
```

Comment:
[CLEANING] You might want to do this instead: 
`const { inject } = require('../context');`